src/main/webapp/WEB-INF/pages/cart.jsp

```diff
@@ -0,0 +1,97 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<jsp:useBean id="cart" type="com.es.phoneshop.model.entity.cart.Cart" scope="session"/>
+<c:set var="servletContextPath" value="${pageContext.servletContext.contextPath}"/>
+<tags:master pageTitle="Cart">
+    <p>
+        Cart: ${cart}
```

Comment:
now no need to display this object in this way, can be removed from al jsp-s
miniCart and cart page is enough